<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XML Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="styleSheet">
         <string notr="true">    background-color: navy;
	color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
	border-color: black;</string>
        </property>
        <property name="text">
         <string>Hide Text Editor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_8">
        <property name="styleSheet">
         <string notr="true">    background-color: navy;
	color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
	border-color: black;</string>
        </property>
        <property name="text">
         <string>Hide Program Text Editor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_2">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="styleSheet">
         <string notr="true">background-color: LightSlateGrey;
color: black;
border-style: outset;
border-width: 1px;
    border-radius: 5px;</string>
        </property>
        <property name="text">
         <string>Open File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="styleSheet">
         <string notr="true">background-color: LightSlateGrey;
color: black;
border-style: outset;
border-width: 1px;
    border-radius: 5px;</string>
        </property>
        <property name="text">
         <string>Compress</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="styleSheet">
         <string notr="true">background-color: LightSlateGrey;
color: black;
border-style: outset;
border-width: 1px;
    border-radius: 5px;</string>
        </property>
        <property name="text">
         <string>Convert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="styleSheet">
         <string notr="true">background-color: LightSlateGrey;
color: black;
border-style: outset;
border-width: 1px;
    border-radius: 5px;</string>
        </property>
        <property name="text">
         <string>Decompress</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="styleSheet">
         <string notr="true">background-color: LightSlateGrey;
color: black;
border-style: outset;
border-width: 1px;
    border-radius: 5px;</string>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="styleSheet">
         <string notr="true">background-color: LightSlateGrey;
color: black;
border-style: outset;
border-width: 1px;
    border-radius: 5px;</string>
        </property>
        <property name="text">
         <string>Correct</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_9">
        <property name="styleSheet">
         <string notr="true">background-color: LightSlateGrey;
color: black;
border-style: outset;
border-width: 1px;
    border-radius: 5px;</string>
        </property>
        <property name="text">
         <string>Format</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>832</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_as"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuXML">
    <property name="title">
     <string>XML</string>
    </property>
    <widget class="QMenu" name="menuConvert_to_JSON">
     <property name="title">
      <string>Convert to JSON</string>
     </property>
     <addaction name="actionCurrent_File"/>
     <addaction name="actionOpen_2"/>
    </widget>
    <addaction name="actionCompress"/>
    <addaction name="menuConvert_to_JSON"/>
    <addaction name="actionDecompress"/>
    <addaction name="actionFormat"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuXML"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCompress">
   <property name="text">
    <string>Compress</string>
   </property>
  </action>
  <action name="actionDecompress">
   <property name="text">
    <string>Decompress</string>
   </property>
  </action>
  <action name="actionDark_Mode">
   <property name="text">
    <string>Dark Mode</string>
   </property>
  </action>
  <action name="actionCurrent_File">
   <property name="text">
    <string>Current File</string>
   </property>
  </action>
  <action name="actionOpen_2">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionFormat">
   <property name="text">
    <string>Format</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
